lib/eslint.js

```diff
@@ -718,11 +720,20 @@ module.exports = (function() {
 
                 severity = getRuleSeverity(config.rules[key]);
                 options = getRuleOptions(config.rules[key]);
+                parserName = config.parser;
+
+                // for use against pre-parsed AST, may be no valid parser configured
+                try {
+                    parser = require(parserName);
```

Comment:
We are already loading the parser here:
https://github.com/benmosher/eslint/blob/parser/lib/eslint.js#L526

We should avoid loading it twice and definitely shouldn't swallow the error.